src/object/cli_obj.c

```diff
@@ -4474,6 +4491,10 @@ dc_obj_update(tse_task_t *task, struct dtx_epoch *epoch, uint32_t map_ver,
 			goto out_task;
 		}
 
+		if (obj_auxi->is_ec_obj && obj_auxi->req_reasbed) {
+			args->iods = obj_auxi->reasb_req.orr_uiods;
+			args->sgls = obj_auxi->reasb_req.orr_usgls;
```

Comment:
obj_rw_req_reassemb() has already done this? Does it mean user parameter will be replaced?